classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -175,68 +179,87 @@ public function render() {
+	 * Display debug info that effects the Elastic Press logs being visible or not.
 	 */
-	protected function display_debug_settings_info() {
+	protected function display_debugging_info() {
 		?>
 		<div class="qm-boxed">
 			<section>
-				<h3><?php esc_html_e( 'Debug Settings', 'debug-bar' ); ?></h3>
+				<h3><?php esc_html_e( 'Debug Info', 'debug-bar' ); ?></h3>
 				<p>
-					<?php esc_html_e( 'One of the following debug settings should be enabled for search query logs to be displayed', 'debug-bar' ); ?>
+					<?php esc_html_e( 'Elastic Press query logs not being displayed can be due to one of the following reasons:', 'debug-bar' ); ?>
 				</p>
 				<table>
 					<tbody>
-					<tr>
 						<?php
-						// WP_DEBUG - is inactive.
-						$status = false;
-						$message = wp_kses(
-							sprintf( __( '%s is either not defined or turned off', 'debug-bar' ), '<code>WP_DEBUG</code>' ),
-							[ 'code' => [] ]
-						);
-
-						// WP_DEBUG - is active.
-						if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-							$status  = true;
-							$message = wp_kses(
-								sprintf( __( '%s is defined', 'debug-bar' ), '<code>WP_DEBUG</code>' ),
-								[ 'code' => [] ]
-							);
-						}
-						?>
-						<td>
-							<?php echo $status ? '&#x2714;' : '&#10060;'; ?>
-						</td>
-						<td>
-							<?php echo $message; ?>
-						</td>
-					</tr>
-					<tr>
+						if ( ! defined( WP_DEBUG ) || ! WP_DEBUG || ! defined( WP_EP_DEBUG ) || ! WP_EP_DEBUG ) :
+							?>
+						<tr>
+							<td><?php echo $this->cross_mark; ?></td>
+							<td>
+								<?php
+								echo wp_kses(
+									sprintf( __( 'Either %s or %s should be enabled', 'debug-bar' ), '<code>WP_DEBUG</code>', '<code>WP_EP_DEBUG</code>' ),
+									[ 'code' => [] ]
+								);
+								?>
+							</td>
+						</tr>
 						<?php
-						// WP_EP_DEBUG - is inactive.
-						$status = false;
-						$message = wp_kses(
-							sprintf( __( '%s is either not defined or turned off', 'debug-bar' ), '<code>WP_EP_DEBUG</code>' ),
-							[ 'code' => [] ]
-						);
-
-						// WP_EP_DEBUG - is active.
-						if ( defined( 'WP_EP_DEBUG' ) && WP_EP_DEBUG ) {
-							$status  = true;
-							$message = wp_kses(
-								sprintf( __( '%s is defined', 'debug-bar' ), '<code>WP_EP_DEBUG</code>' ),
-								[ 'code' => [] ]
-							);
-						}
+						endif;
 						?>
-						<td>
-							<?php echo $status ? '&#x2714;' : '&#10060;'; ?>
-						</td>
-						<td>
-							<?php echo $message; ?>
-						</td>
-					</tr>
+
+						<tr>
+							<?php
+							// EP is currently indexing.
+							if ( ep_is_indexing() ) :
+							// if ( false ) :
+							?>
+							<td><?php echo $this->exclamation_mark; ?></td>
+							<td>
+								<?php esc_html_e( 'Data is being currently indexed. Please try after it is finished', 'debug-bar' ); ?>
+							</td>
+							<?php
+							else :
+							?>
+
+							<td><?php echo $this->check_mark; ?></td>
+							<td>
+								<?php esc_html_e( 'Data has been indexed', 'debug-bar' ); ?>
```

Comment:
I think the message here should be something more like "Index is in currently in sync"